internal/pkg/describe/service.go

```diff
@@ -58,6 +59,7 @@ type configurations []*ServiceConfig
 
 func (c configurations) humanString(w io.Writer) {
 	fmt.Fprintf(w, "  %s\t%s\t%s\t%s\t%s\n", "Environment", "Tasks", "CPU (vCPU)", "Memory (MiB)", "Port")
+	fmt.Fprintf(w, "  %s\t%s\t%s\t%s\t%s\n", strings.Repeat("-", len("Environment")), strings.Repeat("-", len("Tasks")), strings.Repeat("-", len("CPU (vCPU)")), strings.Repeat("-", len("Memory (MiB)")), strings.Repeat("-", len("Port")))
```

Comment:
can we use the new fancy `underline` method here?

Reply:
Doh! Missed this one. Thanks!